test/commands/track/update_build_status_test.rb

```diff
@@ -97,15 +98,15 @@ class Track::UpdateBuildStatusTest < ActiveSupport::TestCase
 
     assert_equal 5, track.build_status.volunteers.num_volunteers
     expected_users = [
-      { rank: 1, activity: '', handle: user_1.handle, reputation: period_1.reputation.to_s,
+      { rank: 1, activity: '', handle: user_1.handle, flair: user_1&.flair, reputation: period_1.reputation.to_s,
```

Comment:
This applies applies to all the other occurences in this file

```suggestion
      { rank: 1, activity: '', handle: user_1.handle, flair: user_1.flair, reputation: period_1.reputation.to_s,
```